### PR TITLE
Add Open Source Collaboration Challenge

### LINKED ISSUE #36: Episode 6: Add discussion about open source software development and the conflict with agile methodologies
At the eScience Center we encountered that starting and contributing to open source software projects leads to many dependencies on collaborators outside the team, the center and the country. Not completely "owning" your product is problematic in many agile methodologies.

### PR DESCRIPTION
**Pull request by raar1**
Fixes #36

Added an extra challenge as suggested.

### CODE REVIEW

File: _episodes/06-agile-in-research.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ and it makes more sense to assign particular tasks to members with a relevant ex
 The goal of the Agile way of working is to maximize value for the customer,
 but what is "value" in research? And who is the customer?
 It is often hard/awkward to define scientific outcomes as products with a certain functionality.
+### Open Source software development challenge
+The nature of open source projects is that they draw (and benefit from) collaborators outside of your team, workplace and even country.
+Starting and contributing to open source software projects therefore leads to many dependencies on collaborators outside of the team.
+Not completely "owning" your product is problematic in many Agile methodologies. Approaches like Scrum may not work well or at all in such a situation.
```
Comment (svenvanderburg) [inline _episodes/06-agile-in-research.md:52]: Nice challenge! Do we have a solution though?

I don't have a lot of experience with this. But my solution would be to have issues like 'Open up an issue on Open Source Project X for problem Y'  and two linked ones: 'Submit pull request on Open Source project B for project X' 'Implement comments on PR Z' In other words make sure that you can commit to the acceptance criteria of the work-item, which in open source projects means splitting investigation, implementation and review process.

Comment (svenvanderburg) [reply _episodes/06-agile-in-research.md:52]: Or maybe making sure that maintainers of the open-source project are aware of the sprint that you are running.

**Review by svenvanderburg — APPROVED** [review]
Nice 👍 It would be nice to add a solution though, see my comments.

File: _episodes/06-agile-in-research.md (hunk 1 above)
Comment (raar1) [reply _episodes/06-agile-in-research.md:52]: Yes, I think you're probably right. The answer is that many/most open source projects clearly do follow a sort of loose Agile process in the use of issues, PRs etc. You need to identify what parts of the project you have "ownership" of, and which parts you don't, and then plan development to mitigate issues. A bit like the PerfectFit project, I suppose? Although goaliejs is obviously not open source.

Comment (svenvanderburg) [reply _episodes/06-agile-in-research.md:52]: Yeah, only the waiting time is predictably unpredictable. If you want, please add this solution to the discussion on solution.

Comment (raar1) [reply _episodes/06-agile-in-research.md:52]: OK I've finally done this. Better late than never...

**Pull request merged**